app/views/application/index.html

```diff
@@ -87,10 +87,10 @@ <h2 class="govuk-heading-m govuk-!-font-size-27">Types of expertise</h2>
       
       
       <section class="govuk-!-margin-bottom-8">
-        <h2 class="govuk-heading-m govuk-!-font-size-27">Areas that you can provide expertise on</h2>
+        <h2 class="govuk-heading-m govuk-!-font-size-27">Subjects you can advise on</h2>
```

Comment:
Sorry, this should be 'Subjects that you can provide expertise on'

Reply:
Thanks. I've made all of the changes suggested below